full_position.hh

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "keys.hh"
+#include "position_in_partition.hh"
+
+struct full_position;
+
+struct full_position_view {
+    const partition_key& partition;
```

Comment:
view?

Reply:
I will attempt to use it although I seem to recall that `partition_key_view` is not a first class citizen when it comes to comparing keys for example. We don't have good views for keys.